sdcm/report_templates/results_base.html

```diff
@@ -187,6 +183,9 @@ <h3>
     {% endif %}
 {% endblock %}
 
+{% block body %}
```

Comment:
why did you move the body block down? this is not part of this PR, please revert